vb_script.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import pennylane as qml
+pl_version = '"' + qml.version() + '"\n' 
+
+
+def bump_version(version_line, pre_release):
+    data = version_line.split(" ")
+    curr_version = data[-1]
+
+    if pre_release:
+        bumped_version = pl_version.replace("-dev", "")  # get current Pennylane version then remove -dev
+#         bumped_version = curr_version.replace("-dev", "")  # remove -dev
+    else:
+        split_version = curr_version.split(".")  # "0.17.0" --> ["0,17,0"]
```

Comment:
Are the elements within `["0,17,0"]` strings? So would this actually be `['"0','17','0"']`?